src/overlays/actors/ovl_Bg_Jya_Ironobj/z_bg_jya_ironobj.c

```diff
@@ -43,21 +59,217 @@ static ColliderCylinderInit D_808994AC = {
     },
     { 30, 150, 0, { 0, 0, 0 } },
 };
+
+static s16 D_808994D8[] = { 0x8700, 0x4000, 0xC000, 0x0000 };
+
+// Pillar
+static s16 D_808994E0[] = { 0x0005, 0x0008, 0x000B, 0x000E, 0x0011, 0x0014, 0x0017, 0x001A };
+static s16 D_808994F0[] = { 0x0012, 0x001A, 0x0022, 0x002A, 0x0032, 0x003C, 0x0046, 0x0050 };
+static s16 D_80899500[] = { 0x0030, 0x002A, 0x0024, 0x0020, 0x001C, 0x0018, 0x0014, 0x0010 };
+
+// Throne
+static s16 D_80899510[] = { 0x0005, 0x0008, 0x000B, 0x000E, 0x0011, 0x0014, 0x0017, 0x001A };
+static s16 D_80899520[] = { 0x0012, 0x001A, 0x0022, 0x002A, 0x0032, 0x003C, 0x0046, 0x0050 };
+static s16 D_80899530[] = { 0x0030, 0x002A, 0x0024, 0x0020, 0x001C, 0x0018, 0x0014, 0x0010 };
+
+static InitChainEntry sInitChain[] = {
+    ICHAIN_VEC3F_DIV1000(scale.x, 100, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneForward, 1000, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneScale, 500, ICHAIN_CONTINUE),
+    ICHAIN_F32(uncullZoneDownward, 1000, ICHAIN_STOP),
+};
+
+static CollisionHeader* sCollisionHeaders[] = { &gPillarCol, &gThroneCol };
+
+void BgJyaIronobj_InitCylinder(BgJyaIronobj* this, GlobalContext* globalCtx) {
+    ColliderCylinder* colCylinder;
+
+    colCylinder = &this->colCylinder;
```

Comment:
```suggestion
    ColliderCylinder* colCylinder = &this->colCylinder;
```
I'm surprised this temp is needed.

Reply:
Some temp var is required to match, so instead of a `pad` I opted for a colCylinder temp.